src/core/grabber/include/grabber/device_grabber_details/entry.hpp

```diff
@@ -153,6 +153,18 @@ class entry final : public pqrs::dispatcher::extra::dispatcher_client {
     }
   }
 
+  void async_start_queue_value_monitor_no_seize(void) {
+    if (hid_queue_value_monitor_) {
+      if (!hid_queue_value_monitor_async_start_called_) {
+        first_value_arrived_ = false;
+        hid_queue_value_monitor_async_start_called_ = true;
+      }
+
+      hid_queue_value_monitor_->async_start(kIOHIDOptionsTypeNone,
```

Comment:
Can add an internal function that gets passed in either `kIOHIDOptionsTypeSeizeDevice` or  `kIOHIDOptionsTypeNone` to avoid code repetition